mantis-server/mantis-server-worker/src/main/java/io/mantisrx/server/worker/ExecuteStageRequestService.java

```diff
@@ -128,18 +157,32 @@ public TrackedExecuteStageRequest call(
                     @Override
                     public void onCompleted() {
                         logger.error("Execute stage observable completed"); // should never occur
-                        executionOperations.shutdownStage();
+                        try {
+                            executionOperations.shutdownStage();
+                        } catch (IOException e) {
```

Comment:
Should you still close cl?

Reply:
CL closure has been added recently. Can you refresh the set of commits?